Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -63,8 +63,8 @@ def edit
              !Homebrew::EnvConfig.no_install_from_api? &&
              !Homebrew::EnvConfig.no_env_hints?
             opoo <<~EOS
-              Unless `HOMEBREW_NO_INSTALL_FROM_API` is set when running `brew install`,
-              it will ignore any locally edited #{(core_cask_path || core_cask_tap) ? "casks" : "formulae"}.
+              `brew install` ignores locally edited #{(core_cask_path || core_cask_tap) ? "casks" : "formulae"} \
+              if `HOMEBREW_NO_INSTALL_FROM_API` is not set.
```

Comment:
I think this was two lines originally because it's more than 80 characters wide and we're trying to be avoid line wrapping here.

Reply:
Reformatted manually to fit 80 char limit in the longest case.